source/includes/_clients.md

```diff
@@ -30,6 +30,19 @@ curl -X POST \
  https://tools.brightlocal.com/seo-tools/api/v1/clients-and-locations/clients/
 ```
 
+### Creating a client
+
+```csharp
+api request = new api("<INSERT_API_KEY>", "<INSERT_API_SECRET>");
+
+var parameters = new api.Parameters();
+       parameters.Add("name", "Le Bernardin");
+       parameters.Add("company-url", "http://www.example.com");
+       parameters.Add("business-category-id", "791");
+
+var success = request.Post("v1/clients-and-locations/clients/", parameters);
```

Comment:
Add leading "/" to all endpoints for consistency. Make sure C# wrapper works with or without.